api/model/storage/postgres/result.go

```diff
@@ -340,7 +335,7 @@ func (s *Storage) PersistResult(dataset string, storageName string, resultURI st
 		}
 		indicesParsed[parsedVal] = true
 
-		dataForInsert := []interface{}{resultURI, parsedVal, targetVariable.Key, records[i][targetIndex]}
+		dataForInsert := []interface{}{resultURI, parsedVal, targetHeaderName, records[i][targetIndex]}
```

Comment:
I'm a bit worried about the switch to header name here. I am not sure what we use that field for, but I would think it makes more sense to assume the value contains the key rather than the header name.

Reply:
I think that's a just a result of me having to merge in the `Key` rename.  Everything works, but I'll replace it with `targetVariable.Key` since that's what gets used almost everywhere else.

Reply:
No, not a result of the refactor, and this is a legit change.  `targetHeaderName` is generated through a function `getHeaderName` that will return `targetVariable.Key` for a normal variable, or will chase down the appropriate target column in the raw data for grouped variables.  The current example of this is timeseries - the target is the top compound timeseries variable, but the actual value that is predicted at the TA2 level is the the YCol (value) field of the grouped variable.